code/modules/antagonists/cult/cult_structures.dm

```diff
@@ -179,13 +179,25 @@
 	light_color = COLOR_SOFT_RED
 	break_sound = 'sound/effects/glassbr2.ogg'
 	break_message = "<span class='warning'>The blood-red crystal falls to the floor and shatters!</span>"
-	var/heal_delay = 25
```

Comment:
wouldnt this mean you made cult pylons heal 25/3 times faster since hte subsystem processes every 3 deciseconds

Reply:
wait i missed the cooldown